docs/features/asc1/stateless/sdks.md

```diff
@@ -257,12 +262,18 @@ The SDKs require that parameters to a stateless smart contract TEAL program be i
     The samples show setting parameters at the creation of the logic signature. These parameters can be changed at the time of submitting the transaction.
 
 ```javascript tab="JavaScript"
-    //string parameter
-    let args = ["my string"];
-    let lsig = algosdk.makeLogicSig(program, args);
-    //integer parameter
-    let args = [[123]];
-    let lsig = algosdk.makeLogicSig(program, args);
+    // string parameter
+    const args = ["my string"];
+    const lsig = algosdk.makeLogicSig(program, args);
```

Comment:
@ryanRfox was just making these update and retesting things and I'm not sure this actually works. I tried a basic program with just a simple secret word. 

```
#pragma version 2
arg_0
byte "xxx"
==
```
Call it like this:

```
const args = ["xxx"]
    const lsig = algosdk.makeLogicSig(program, args);
```
And it's always rejected by logic. Am I missing something basic? I can get the int example working but not this.

Reply:
I got it to work like this:

```const arg = [...Buffer.from('xxx')];

    const args = [arg];
    const lsig = algosdk.makeLogicSig(program, args);
```
Should we do it like this or recommend something else?

Reply:
Good catch here as well. Another example for your consideration is located at [application call with arguments](https://developer.algorand.org/docs/features/asc1/stateful/sdks/#call-with-arguments) (which also requires refactoring in a separate PR). It uses `Buffer.from` as we don't yet have an encoder helper for strings. I prefer to include the `appArgs.push` method (even if only adding a single element) to demonstrate that pattern.  

```
// call application with arguments
let ts = new Date(new Date().toUTCString());
console.log(ts)
let appArgs = [];
appArgs.push(new Uint8Array(Buffer.from(ts)));
```